query_execution/WorkOrdersContainer.hpp

```diff
@@ -142,7 +142,7 @@ class WorkOrdersContainer {
    *         available, return nullptr.
```

Comment:
We need to add comments about the ownership-semantics here. It will be good if we explain the flow of ownership : Relational Operators produce the WorkOrders, the container owns it until it is scheduled. Once the WorkOrder is scheduled, the ownership is passed to the Worker that executes it and the executing worker deletes it.